Video.js

```diff
@@ -114,12 +114,6 @@ export default class Video extends Component {
     }
   };
 
-  _onProgress = (event) => {
-    if (this.props.onProgress) {
-      this.props.onProgress(event.nativeEvent);
```

Comment:
by removing this you are breaking the argument passed on onProgress callback.

Before : event.nativeEvent
After: event

This will break every onProgress code.